app/controllers/steps_controller.rb

```diff
@@ -9,18 +9,25 @@ def create
   end
 
   def create_confirmation
+    #AMY EDIT THIS ACTION
     @current_user = User.find_by_id(current_user)
     @tutorial = Tutorial.find_by_id(params[:tutorial_id])
     @chapter = Chapter.find_by_id(params[:chapter_id])
-    @new_step = Step.new(title: params[:step][:title], description: params[:step][:description], draft: params[:step][:draft], warning: params[:step][:warning], start_time: (params[:step][:start_time]).to_i, end_time: (params[:step][:end_time]).to_i, chapter_id: @chapter.id)
-    if @new_step.valid?
-      @new_step.save
-      @chapter.set_chapter_start_and_end_time_based_on_step_times
+    
+    params[:chapter][:steps_attributes].each do |key, value|
```

Comment:
Unused block argument - `value`. If it's necessary, use `_` or `_value` as an argument name to indicate that it won't be used.